src/App.jsx

```diff
@@ -21,11 +22,13 @@ function App() {
         path="/UserDashboard"
         element={<UserDashboard />}
```

Comment:
should also pass user in as props here

Reply:
i'm not implementing the user dashboard, but maybe bring this up to person responsible?